app/src/pages/landing/LandingPage.tsx

```diff
@@ -1,15 +1,178 @@
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import clsx from 'clsx';
+import { AuthGuard, UnAuthGuard } from 'components/security/Guards';
+import { SYSTEM_ROLE } from 'constants/roles';
 import { AuthStateContext } from 'contexts/authStateContext';
-import React, { useContext, useEffect } from 'react';
-interface ILandingPageProps {
-  originalPath: string;
-}
+import BaseLayout from 'layouts/BaseLayout';
+import React, { useContext, useMemo } from 'react';
+import { Link } from 'react-router-dom';
 
-export const LandingPage: React.FC<ILandingPageProps> = ({ originalPath }) => {
+const useStyles = makeStyles((theme: Theme) => ({
+  baseLayoutContainer: {
+    // Contingency background, pending hero image load
+    background: '#00438A linear-gradient(to bottom, #00438A, #00274D)',
+    backgroundImage: `url('/assets/hero.jpg')`,
+    backgroundPosition: 'center',
+    backgroundRepeat: 'no-repeat',
+    backgroundSize: 'cover',
+    color: theme.palette.primary.contrastText,
+    minHeight: '100vh',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    height: '30rem',
+
+    '& > main': {
+      display: 'flex',
+      marginTop: theme.spacing(-6)
+    }
+  },
+  heroHeader: {
+    maxWidth: '22ch',
+    fontSize: '3.5rem',
+    letterSpacing: '-0.03rem',
+    textShadow: '0px 0px 15px rgba(0,13,26,0.5)'
+  },
+  heroSubheader: {
+    maxWidth: '45ch',
+    margin: '2rem 0 4rem',
+    fontSize: '1.325rem',
+    lineHeight: '1.5',
+    textShadow: '0px 0px 10px rgba(0,13,26,1)'
+  },
+  greeting: {
+    fontSize: '1.125rem',
+    lineHeight: '1.5'
+  },
+  heroButton: {
+    color: theme.palette.primary.main,
+    backgroundColor: '#fcba19',
+    fontWeight: 700,
+    padding: '12px 44px'
+  },
+  loginButton: {
+    textTransform: 'uppercase'
+  },
+  actionsContainer: {
+    maxWidth: '45ch',
+
+    '& p': {
+      margin: '1.5rem 0 1rem'
+    }
+  },
+  actions: {
+    display: 'flex',
+    flexFlow: 'row nowrap',
+    alignItems: 'center',
+    gap: '1rem',
+
+    '& > span': {
+      textTransform: 'uppercase'
+    }
+  },
+  heroLink: {
+    color: '#fcba19',
+    fontWeight: 700,
+    textDecoration: 'none'
+  },
+  username: {
+    textTransform: 'uppercase'
+  }
+}));
+
+export const LandingPage = () => {
   const { keycloakWrapper } = useContext(AuthStateContext);
+  const classes = useStyles();
+
+  const loginUrl = useMemo(() => keycloakWrapper?.getLoginUrl(), [keycloakWrapper]);
+  const userIdentifier = keycloakWrapper?.getUserIdentifier() || '';
 
-  useEffect(() => {
-    keycloakWrapper?.keycloak?.login();
-  });
+  const hasAdministrativeRole = keycloakWrapper?.hasSystemRole([
+    SYSTEM_ROLE.DATA_ADMINISTRATOR,
+    SYSTEM_ROLE.SYSTEM_ADMIN
+  ]);
 
-  return null;
+  return (
+    <BaseLayout className={classes.baseLayoutContainer}>
+      <Container maxWidth="md">
```

Comment:
We're going to have to come up with a better strategy here - this layout seems odd on larger viewports.

Reply:
Encapsulate the content with a negative top margin to align visual center here, not at the containing element.